aries_cloudagent/ledger/indy.py

```diff
@@ -486,95 +485,108 @@ def __init__(
 
         """
 
+        async def cred_def_in_wallet(wallet_handle, cred_def_id) -> bool:
```

Comment:
Could this function be useful on its own? Does it make sense to move it out to the class level?

Reply:
No: it is useful only in establishing whether this issuer can use this cred def already. If it were useful at class level, it would go in the indy wallet, not the indy ledger. Once the cred defs per issuer are established, the non-secrets records already serve to tell what cred defs are available, but this tiny call helps bootstrap those into place.